### PR TITLE
P#48307 Remove backend data types from configuration mask that do not make sense

### PR DESCRIPTION
**Pull request by hungnc89**

### CODE REVIEW

File: plugin/Field/Collection/FieldLoaderGeneric.php [hunk 1]
```diff
@@ -80,9 +80,9 @@ public function load(): Generator
 			if (isset($fieldArray['label'])) {
 				unset($fieldArray['label']);
 			}
-
+			$listTypeUnSupported = ['user', 'datei', 'redhint', 'blackhint', 'dividingline'];
 			foreach ($fieldArray as $fieldName => $fieldProperties) {
-				if ($module === onOfficeSDK::MODULE_ADDRESS && $fieldProperties['tablename'] === 'addressFaktura') {
```
Comment (jayay) [inline plugin/Field/Collection/FieldLoaderGeneric.php:85]: Is there a reason why this condition got removed? I think this is still necessary

Comment (jayay) [reply plugin/Field/Collection/FieldLoaderGeneric.php:85]: Thanks

**Review by jayay — APPROVED** [review]

**Comment by vu-vuong-esg** [comment]
Hi @jayay, I updated new code to fix task #48307, can you take a look

**Review by jayay — APPROVED** [review]
LGTM 👍🏻

**Comment by jayay** [comment]
I think something went wrong with the merge. 47 modified files seems wrong.

**Comment by jayay** [comment]
Turned this into a draft until it is resolved.

**Review by jayay — APPROVED** [review]

**Comment by michaelarnold-dev** [comment]
belongs to P#48307

**Pull request merged**